amplify/data/resource.ts

```diff
@@ -8,16 +8,27 @@ enum TemplateType {
   LETTER = 'LETTER',
 }
 
+enum TemplateStatus {
+  NOT_YET_SUBMITTED = 'NOT_YET_SUBMITTED',
+  SUBMITTED = 'SUBMITTED',
+}
+
 const templateTypes = [
   TemplateType.NHS_APP,
   TemplateType.SMS,
   TemplateType.EMAIL,
   TemplateType.LETTER,
 ] as const;
 
+const templateStatuses = [
+  TemplateStatus.NOT_YET_SUBMITTED,
+  TemplateStatus.SUBMITTED,
+];
```

Comment:
Really minor. Can you add `as const` - should prevent the array from being modified externally.

Reply:
updated